plugins/system/kernel_same_page_merging

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+#%# family=auto
+
+: << EOF
+=head1 NAME
+kernel_same_page_merging - Outputs various metrics about same page merging.
+
+=head1 AUTHOR
+=over 4
+=item * Michael Grote
+=back
+
+=head1 LICENSE
+GPLv3 or later
+
+SPDX-License-Identifier: GPL-3.0-or-later
+
+=head1 MAGIC MARKERS
+=begin comment
+These magic markers are used by munin-node-configure when installing
+munin-node.
+=end comment
+  #%# family=auto
+=cut
+EOF
+#
+
+# Variablen
+ksm_dir_run="/sys/kernel/mm/ksm/run"
+
+function labels {
```

Comment:
Personally I prefer a more verbose naming style for functions and variables. Maybe `print_config`? But this is just a matter of taste ...